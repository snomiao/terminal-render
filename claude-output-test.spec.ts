import { readFile } from "fs/promises"
import { TerminalTextRender } from "."

it('works', async () => {
    const input = await readFile('./claude-code-execute-log.txt', 'utf8')
    const out = new TerminalTextRender().write(input).render()
    expect(out).toMatchSnapshot()
})