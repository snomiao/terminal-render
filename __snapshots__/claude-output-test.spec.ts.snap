// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`works 1`] = `
"claude-code-execute@1.0.0:  do nothing and exit
⭐ Starting claude, automatically responding to yes/no prompts...
⚠️ Important Security Warning: Only run this on trusted repositories. This tool automatically responds to prompts and can execute commands without user confirmation. Be aware of potential prompt injection attacks where malicious code or instructions could be embedded in files or user inputs to manipulate the automated responses.
╭─────────────────────────────────────────────────────╮
│ ✻ Welcome to Claude Code!                           │
│                                                     │
│   /help for help, /status for your current setup    │
│                                                     │
│   cwd: /code/snomiao/claude-code-execute/tree/main  │
╰─────────────────────────────────────────────────────╯

> do nothing and exit

● I'll exit without taking any action.


> /exit 
  ⎿  (no content)

╭──────────────────────────────────────────────────────────────────────────────╮
│ >                                                                            │
╰──────────────────────────────────────────────────────────────────────────────╯
  ? for shortcuts                                           ◯ IDE disconnected

Total cost:            $0.0171
Total duration (API):  2.7s
Total duration (wall): 48.3s
Total code changes:    0 lines added, 0 lines removed
Usage by model:
       claude-sonnet:  4 input, 14 output, 10.8k cache read, 3.6k cache write

With the $100/mo Max plan, use Sonnet as your daily driver with predictable pricing. • /upgrade to sign up"
`;
